README.md

```diff
@@ -56,11 +57,16 @@ GitHub Action for automatically syncing LeetCode submissions to a GitHub reposit
 
 ## Inputs
 
+- `destination-folder` *(optional)*: The folder in your repo to save the submissions to (necessary for shared repos)
```

Comment:
nit: Could you move this below the required arguments? (and maybe also add *(optional)* to filter-duplicate-secs